service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1254,31 +1267,89 @@ public void deleteOrder(Long id) {
     }
 
     @Override
-    public FondyOrderResponse processOrderFondyClientForIF(OrderFondyClientDto dto) throws Exception {
-        Order order = orderRepository.findById(dto.getOrderId()).orElseThrow();
+    public FondyOrderResponse processOrderFondyClientForIF(OrderFondyClientDto dto, String uuid) throws Exception {
```

Comment:
this method seems to be very complex. please refactor it by extracting some parts into private methods